src/config-loader/config-loader.ts

```diff
@@ -282,14 +310,36 @@ export const setup = (app: Express.Application) => {
 				}),
 			);
 			await loadConfig(configObj);
+			await loadAsyncMigrations(configObj);
 		} catch (err) {
 			console.error('Error loading application config', err, err.stack);
 			process.exit(1);
 		}
 	};
 
+	const loadAsyncMigrations = async (data: Config): Promise<any> => {
```

Comment:
This isn't just loading the migrations -- it executes them as well.

```suggestion
	const runAsyncMigrations = async (data: Config): Promise<any> => {
```

Reply:
Thx!